src/components/v5/common/ActionSidebar/partials/AmountField/hooks.ts

```diff
@@ -30,10 +30,11 @@ export const useAmountField = (selectedTokenAddress: string) => {
 
   const onInput = (event: React.ChangeEvent<HTMLInputElement>) => {
     const { value } = event.target;
-    const valueWithoutCommas = value.replace(/,/g, '');
-    const width = valueWithoutCommas.length * 0.65;
-
-    setInputWidth(width);
+    if (/^[\d,]*$/.test(value)) {
```

Comment:
Can that regex be stored in a meaningfully named variable or a comment be added to what it checks for? It's a bit hard to tell on first look